lib/function.g

```diff
@@ -159,6 +159,31 @@ BIND_GLOBAL( "TYPE_OPERATION",
 ##
 DeclareOperationKernel("NameFunction", [IS_OBJECT], NAME_FUNC);
 
+#############################################################################
+##
+#O  NameFunction( <func> )  . . . . . . . . . . . . . . .  name of a function
+##
+##  <#GAPDoc Label="HasNameFunction">
```

Comment:
So you add `<#GAPDoc Label="HasNameFunction">` but this is not linked from anywhere in the manual - is this intended?